app/views/users/index.html.erb

```diff
@@ -1,18 +1,17 @@
-<!DOCTYPE html>
-<html>
-<head>
-  <title>Users List</title>
-</head>
-<body>
-<h1>Users List</h1>
-<ul>
-  <% if @users.empty? %>
-    <li>Here is a list of users</li>
-  <% else %>
-    <% @users.each do |user| %>
-      <li><%= link_to user.name, user_path(user) %></li>
+<h1>Users#index</h1>
+
+<section>
+  <% @users.each do |user| %>
+    <%= link_to user_path(user.id) do %>
+      <div class="user">
```

Comment:
- [Optional] Dear @theresetuyi, In such case when you have to render a list of things/users it would be wise to use partials, this will make your code more readable and reusable.  Something like `<%= render @users %>`